### PR TITLE
docs(aio): add Material Community Components

### PR DESCRIPTION
**Pull request by tiaguinho**
@StephenFluin can you check this?

### CODE REVIEW

**Review by StephenFluin — APPROVED** [review]
From a content perspective this looks good. Not sure why CircleCI is failing though, so we need to figure that out.

**Comment by kara** [comment]
@StephenFluin The failure is unrelated to this PR. It was a rebase error that is now fixed in master

**Comment by tiaguinho** [comment]
Thanks, @kara.
Is everything alright then?

**Comment by tiaguinho** [comment]
@StephenFluin, do you think I need to do some change on this PR?

**Comment by StephenFluin** [comment]
It looks like you added another commit. We'll need to squash / remove that before we can merge.

**Comment by tiaguinho** [comment]
Hi @StephenFluins sorry for that. I think it's correct now.

**Review by StephenFluin — CHANGES_REQUESTED** [review]
One tiny problem still, can you add back the newline at the end of the file so there isn't the extra change?

**Comment by tiaguinho** [comment]
Just add the line. Can I do anything more to help?

**Comment by gkalpak** [comment]
@tiaguinho, you just need to squash the commits and we're good to go :grin:

**Comment by tiaguinho** [comment]
@gkalpak I think we good to go now... 😃

File: aio/content/marketing/resources.json [hunk 1]
```diff
@@ -704,3 +711,4 @@
     }
   }
 }
+
```
Comment (gkalpak) [inline aio/content/marketing/resources.json]: Where did this come from? can you remove it please :grin:

**Pull request closed without merging**